pkg/cloudprovider/azure.go

```diff
@@ -380,6 +432,47 @@ func (a *Azure) getNetworkInterfaces(instance *compute.VirtualMachine) ([]networ
 	return networkInterfaces, nil
 }
 
+func splitObjectID(azureResourceID string) (string, string, string) {
```

Comment:
NIT this is just a suggestion:
```
func splitObjectID(azureResourceID string) (resourceGroupName, loadBalancerName, backendAddressPoolName string) {
```
and then remove line 443 and 460 is just `return` because we know what these returning strings are :)

Reply:
sure!